### ISSUE TITLE
Bug: sbb-select accessibility issue with Safari & macOs 

### ISSUE THREAD

**Issue by IForgotAccount** [sbb-design-systems/sbb-angular#1613]
**Describe the bug**
There is an accessibility issue with the Safari browser on macOs. 

The sbb-select does not read out with the screen reader VoiceOver the entries selected or ticked by arrow key. 

**To Reproduce**
Use macOs 15.4 or 15.5 with Safari 15.5

1, Start VoiceOver (Settings->Accessibility)
2. Go to https://angular.app.sbb.ch/angular/components/select/examples
3. Scroll down to 'Select Option Groups Multi Selection'
4. Tab to 'sbb-select International food'
5. Open sbb-select
6. Try select some option ex. Pasta
7. Error: Listen missing sound output 

**Expected behavior**
Same sound output like Chrome/Firefox on WIndows.

**Desktop**
 - OS: macOs 15.5
 - Browser safari
 - Version 15

**Comment by mhaertwig** [comment]
Sorry, I closed this accidentally.

**Comment by mhaertwig** [comment]
I could not find any solution for this bug. And I also did not find any MultiSelect widget that consists of a `combobox` and a `listbox` (like ours) that works well with VoiceOver and Safari. Even the [W3 example](https://www.w3.org/WAI/ARIA/apg/example-index/combobox/grid-combo.html) is not working properly.

There are several Webkit bug reports that seem to be related to this issue, e.g.
* https://bugs.webkit.org/show_bug.cgi?id=231724
* https://bugs.webkit.org/show_bug.cgi?id=167671

And there are bug reports for other libraries like e.g. Angular Components: https://github.com/angular/components/issues/24390

**Comment by mhaertwig** [comment]
I'll keep an eye on the linked issues and update this bug if there's a solution.
